src/cdomain/value/cdomains/intDomain.ml

```diff
@@ -572,6 +572,14 @@ module IntervalArith (Ints_t : IntOps.IntOps) = struct
     let min_ik' = Ints_t.to_bigint min_ik in
     let t = List.find_opt (fun x -> Z.compare l x >= 0 && Z.compare x min_ik' >= 0) ts in
     BatOption.map_default Ints_t.of_bigint min_ik t
+  let is_upper_threshold u =
+    let ts = if get_interval_threshold_widening_constants () = "comparisons" then WideningThresholds.upper_thresholds () else ResettableLazy.force widening_thresholds in
+    let u = Ints_t.to_bigint u in
+    List.exists (fun x -> Z.compare u x = 0) ts
```

Comment:
```suggestion
    List.exists (Z.equal u x) ts
```

Reply:
It would have to be just `(Z.equal u)` then.

Reply:
True!

Reply:
Done!